.github/workflows/daily-rhel-copr.yml

```diff
@@ -9,16 +9,31 @@ on:
 jobs:
   build:
     name: copr-builder
-    runs-on: [self-hosted, ci-tasks, rhel-8]
+    runs-on: [self-hosted, kstest]
+    env:
+      CI_CONTAINER: rhinstaller/anaconda-rhel-copr:rhel-8
     steps:
-      - name: Checkout copr-builder repository
+      - name: Clone anaconda repository
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          fetch-depth: 0
+          path: anaconda
+
+      - name: Build the build container
+        run: |
+          sudo podman build --no-cache -t ${{ env.CI_CONTAINER }} anaconda/dockerfile/anaconda-rhel-copr
+
+      - name: Clone copr-builder repository
         uses: actions/checkout@v2
         with:
           repository: vojtechtrefny/copr-builder
+          fetch-depth: 0
+          path: copr-builder
```

Comment:
I think moving the copr builder into the container will be better solution. At the end it's a dependency and also it will make debugging in the future simpler.